cmd/kube-proxy/app/server.go

```diff
@@ -399,6 +399,11 @@ func NewProxyServer(config *componentconfig.KubeProxyConfiguration, cleanupAndEx
 	proxyMode := getProxyMode(string(config.Mode), iptInterface, iptables.LinuxKernelCompatTester{})
 	if proxyMode == proxyModeIPTables {
 		glog.V(0).Info("Using iptables Proxier.")
+		nodeIP := net.ParseIP(config.BindAddress)
+		if local, _ := isLocalIP(nodeIP); !local {
```

Comment:
L402 would not return nil, see: L128, we enforce user input a valid ip, and on https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/componentconfig/v1alpha1/defaults.go#L56,  bind-address have default value `0.0.0.0`.

I don't know if bind-address is specified by user, maybe need check if bind-address equal `0.0.0.0`?